### PR TITLE
Init Local Storage with Typescript

### PR DESCRIPTION
**Pull request by aahenry51**

### CODE REVIEW

File: mobile/src/utilities/localStorage.type.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export type TlocalStorageProps = {
+  key?: string;
```
Comment (raynesmike) [inline mobile/src/utilities/localStorage.type.ts:2]: I think we should make key a required since were always need the keys to get, set, or delete it

**Review by raynesmike — APPROVED** [review]

**Review by leo-he-1 — APPROVED** [review]

File: mobile/src/utilities/localStorage.type.ts (hunk 1 above)
Comment (aahenry51) [reply mobile/src/utilities/localStorage.type.ts:2]: got it

Comment (aahenry51) [reply mobile/src/utilities/localStorage.type.ts:2]: I think the value should be any too.

Comment (aahenry51) [reply mobile/src/utilities/localStorage.type.ts]: like value: any

**Review by Dulaya — APPROVED** [review]

**Pull request merged**